modules/web-mercator/test/spec/fit-bounds.spec.js

```diff
@@ -3,7 +3,7 @@ import {fitBounds} from '@math.gl/web-mercator';
 import {WebMercatorViewport} from '@math.gl/web-mercator';
 import {toLowPrecision} from '../utils/test-utils';
 
-const FITBOUNDS_TEST_CASES = [
+const FITBOUNDS_TEST_CASES = /** @type {[import('@math.gl/web-mercator/fit-bounds').FitBoundsOptions, import('@math.gl/web-mercator/fit-bounds').Bounds][]} */ ([
```

Comment:
We usually do a separate section with `@typedefs` above, so we can just do `/** @type */` here, but that can be fine tuned later.